### DISCUSSION TITLE
Time based webhooks

### DISCUSSION THREAD

**Discussion by marekdziem** [AzuraCast/AzuraCast#6917] [Ideas]
Hi Azura-casters!

Is it possible to implement webhooks that fire purely based on specified times? 
These would be used to update a station's currently playing show data - eg. a breakfast show, late night jazz show etc.
For a traditional station that's feeding in a single live encoder stream, the new live streamer hook is redundant for this purpose.

Alternatively fire webhook based on scheduled playlist change?

**Comment by Moonbase59** [comment]
On playlist change, and giving playlist name and description would actually be nice.
